ngz/projects/ngz-utils-identity/src/lib/services/auth/Authentication/index.ts

```diff
@@ -51,60 +57,79 @@ export class AuthenticationService {
   /**
    * API auth access token creation time
    */
-  public get accessTokenCTime(): number | null {
+  private get _accessTokenCTime(): number | undefined {
     const value = localStorage.getItem('ngz-utils-identity:accessTokenCTime');
-    return value ? parseInt(value, 10) : null;
+    return value ? parseInt(value, 10) : undefined;
   }
-  public set accessTokenCTime(value: number | null) {
+  private set _accessTokenCTime(value: number | undefined) {
     if (value) {
-      localStorage.setItem(
-        'ngz-utils-identity:accessTokenCTime',
-        value.toString()
-      );
+      localStorage.setItem('ngz-utils-identity:accessTokenCTime', value.toString());
     } else {
       localStorage.removeItem('ngz-utils-identity:accessTokenCTime');
     }
   }
   /**
    * API auth access token time-to-live
    */
-  public get accessTokenTTL(): number | null {
+  private get _accessTokenTTL(): number | undefined {
     const value = localStorage.getItem('ngz-utils-identity:accessTokenTTL');
-    return value ? parseInt(value, 10) : null;
+    return value ? parseInt(value, 10) : undefined;
   }
-  public set accessTokenTTL(value: number | null) {
+  private set _accessTokenTTL(value: number | undefined) {
     if (value) {
-      localStorage.setItem(
-        'ngz-utils-identity:accessTokenTTL',
-        value.toString()
-      );
+      localStorage.setItem('ngz-utils-identity:accessTokenTTL', value.toString());
     } else {
       localStorage.removeItem('ngz-utils-identity:accessTokenTTL');
     }
   }
   /**
    * API auth refresh token if onw is known
    */
-  public get refreshToken(): string | null {
-    return localStorage.getItem('ngz-utils-identity:RefreshToken');
+  private get _refreshToken(): string | undefined {
+    return localStorage.getItem('ngz-utils-identity:RefreshToken') || undefined;
   }
-  public set refreshToken(value: string | null) {
+  private set _refreshToken(value: string | undefined) {
     if (value) {
       localStorage.setItem('ngz-utils-identity:RefreshToken', value);
     } else {
       localStorage.removeItem('ngz-utils-identity:RefreshToken');
     }
   }
 
+  /**
+   * Holds reference to token refresh timeout
+   */
+  private _refreshTokenTimeout?: any;
+
   /**
    * Authenticated user info
    */
   private _info?: any;
   /**
    * Gets info for the authenticated user or false if user not authenticated
```

Comment:
Gets info for the authenticated user or **undefined** if user not authenticated?